src/routes/open/components/ReviewInformation/index.jsx

```diff
@@ -148,7 +162,12 @@ const ReviewComponent = ({ values, classes, network }: Props) => {
       <Row className={classes.info} align="center">
         <Paragraph noMargin color="primary" size="md">
           You&apos;re about to create a new Safe and will have to confirm a transaction with your currently connected
-          wallet. Make sure you have ETH in this wallet to fund this transaction.
+          wallet. The creation will cost approximately
```

Comment:
This is not the text that appears in the designs

Reply:
we changed it with @lukasschor